vignettes/baselinenowcast.Rmd

```diff
@@ -431,26 +432,72 @@ exp_obs_nowcasts <- add_obs_errors_to_nowcast(
 ```
 
 Next, we will take the list of expected observed reporting squares and
-convert this into a long tidy dataframe, indexed by reference time `t` and
-delay `d`, and plot the results
+convert this into a long tidy dataframe, indexed by reference time `t`,
+delay `d`, and draws.
```

Comment:
Bleed from the other PR.

Reply:
I think I intentionally changed this bc we no longer plot the results of that particular step here...